nifi-nar-bundles/nifi-ignite-bundle/nifi-ignite-processors/src/main/java/org/apache/nifi/processors/ignite/AbstractIgniteProcessor.java

```diff
@@ -92,6 +94,14 @@ public void initializeIgnite(ProcessContext context) {
             return;
         }
 
+        List<Ignite> grids = Ignition.allGrids();
+
+        if ( grids.size() == 1 ) {
+            getLogger().warn("Ignite grid already avaialble");
```

Comment:
I think this should not be a warning because if using Put and Get processors on the canvas we will get a warning bulletin although there is no issue. Maybe log as info and inform the user we use a shared grid? (also take care of the typo).